app/Phragile/RenderedTask.php

```diff
@@ -0,0 +1,119 @@
+<?php
+namespace Phragile;
+
+class RenderedTask {
```

Comment:
Unhelpful comment: Looking at the `Task` class I found it a bit annoying how similar these two are. I see that they each have their purpose but something does not seem right about this.

I will finish scanning the entire PR and try to come up with a more constructive comment/suggestion.